package.json

```diff
@@ -240,7 +240,7 @@
     "webpack-cli": "4.7.2",
     "webpack-dev-middleware": "4.2.0",
     "webpack-dev-server": "4.0.0-rc.0",
-    "webpack-hot-middleware": "2.25.0",
+    "webpack-hot-middleware": "2.25.1",
```

Comment:
This was a minor dedupe related to the storybook upgrade (new version of storybook requests 2.25.1, and updating our version to match seemed very low risk)